spec/features/agents/rdv_wizard/step3_spec.rb

```diff
@@ -0,0 +1,62 @@
+RSpec.describe "Step 3 of the rdv wizard" do
+  let(:motif) { create(:motif, :by_phone, organisation: organisation) }
+  let(:default_params) do
+    {
+      organisation_id: organisation.id,
+      duration_in_min: 30,
+      motif_id: motif.id,
+      starts_at: 2.days.from_now,
+      step: 3,
+    }
+  end
+  let!(:user) do
+    create(:user, organisations: [organisation], first_name: "Francis", last_name: "Factice")
+  end
+  let(:other_territory) { create(:territory) }
+  let(:organisation) { create(:organisation) }
+  let(:organisation_from_other_territory) { create(:organisation, territory: other_territory) }
+  let(:user_from_other_territory) do
+    create(:user, organisations: [organisation_from_other_territory], first_name: "Gaston", last_name: "Bidon")
+  end
+
+  let(:agent) { create(:agent, service: motif.service, basic_role_in_organisations: [organisation]) }
+
+  before do
+    login_as(agent, scope: :agent)
+  end
+
+  context "when passing a user_id from a different territory" do
+    let(:params) do
+      default_params.merge(user_ids: [user_from_other_territory.id])
+    end
+
+    it "raises an error and doesn't displays the user info" do
+      visit new_admin_organisation_rdv_wizard_step_path(params)
+      expect(page).not_to have_content(user_from_other_territory.full_name)
```

Comment:
on pourrait peut-être ajouter un petit expect positif ici comme 

```suggestion
      expect(page).to have_content("Vous n’avez pas les permissions nécessaires")
```